.github/workflows/publish_to_wiki.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish to Wiki
+
+# Trigger this action only if there are changes pushed to the docs/** directory under the main branch
+on:
+  push:
+    paths:
+      - markdown/wiki/** # This includes all sub folders
+    branches:
+      - main # This can be changed to any branch of your preference
+
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }} # This is the repository secret
+  USER_NAME: <USERNAME> # Enter the username of your (bot) account
+  USER_EMAIL: <EMAIL> # Enter the e-mail of your (bot) account
+  OWNER: ${{ github.event.repository.owner.name }} # This is the repository owner
+  REPOSITORY_NAME: ${{ github.event.repository.name }} # This is the repository name
+
+jobs:
+  publish_to_wiki:
+    name: Publish to Wiki
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install podman
+        run: |
+          apt-get -qq -y podman
+      # 1. Create folder named `tmp_wiki`
+      # 2. Initialize Git
+      # 3. Pull old Wiki content
+      - name: Pull content from wiki
+        run: |
+          mkdir tmp_wiki
+          cd tmp_wiki
+          git init
+          git config user.name $USER_NAME
+          git config user.email $USER_EMAIL
+          git pull https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git
```

Comment:
This is slightly confusing, is there a reason you cannot use [`actions/checkout`](https://github.com/actions/checkout) for this?  It accepts a custom token for cloning.  I think we even already have a towen setup with write-access under the `rh-container-bot` account.  Details are in the team's shared password manager app.  If you don't have access, ping me and I'll add you.